content/coronavirus_education_page.yml

```diff
@@ -15,15 +15,17 @@ content:
           url: https://www.nidirect.gov.uk/articles/coronavirus-covid-19-advice-schools-colleges-and-universities
     list_heading: "In England:"
     list:
+      - twice-weekly testing for pupils and staff, for their households, and for their support and childcare bubbles
       - all pupils return to schools on 8 March 
-      - GCSE, AS and A level exams will not go ahead as planned
       - university students on practical courses who need to access specialist facilities and equipment can return to in-person teaching from 8 March
   guidance_section:
     header: Popular content
     list:
-      - text: Parents and carers - what you need to know about the return to schools 
-        url: https://www.gov.uk/government/publications/what-parents-and-carers-need-to-know-about-early-years-providers-schools-and-colleges-during-the-coronavirus-covid-19-outbreak
-      - text: Schools - what you need to know about the return to schools 
+      - text: Get twice-weekly tests if you live with, or are in a support or childcare bubble with, a school pupil or school staff
+        url: /guidance/rapid-lateral-flow-testing-for-households-and-bubbles-of-school-pupils-and-staff?priority-taxon=b350e61d-1db9-4cc2-bb44-fab02882ac25  
+      - text: Parents and carers: what you need to know
```

Comment:
> Parents and carers: what you need to know

This will need to be:
`Parents and carers - what you need to know` as the `:` will be seen as a new entry